src/Interceptor/Impl/SecurityInterceptor.php

```diff
@@ -31,6 +39,11 @@ public function prefix(Instance $instance): Response
         $parameters = $instance->getMethod()
             ->getParameters()
         ;
+
+        if ($this->config?->bypassSecurity) {
+            return new Response();
+        }
```

Comment:
Are we sure we're not creating a backdoor usable larger than we expect ?

Reply:
there two things to secure against this in the other PR :

`app.php` which is used in prod, always forces bypass env var to 0
only service proxy 
config is only loaded in `dev/openclassrooms_service_proxy.yml` which is loaded only in dev